packages/backend/src/config/parseConfig.ts

```diff
@@ -419,6 +419,12 @@ const mergeWithEnvironment = (config: LightdashConfigIn): LightdashConfig => {
             release: VERSION,
             environment:
                 process.env.NODE_ENV === 'development' ? 'development' : mode,
+            tracesSampleRate: parseFloat(
```

Comment:
Additional error handling is needed, e.g. `getIntegerFromEnvironmentVariable` https://github.com/lightdash/lightdash/blob/1869560b80b22d0b6c950ef71a63d42a809cfa86/packages/backend/src/config/parseConfig.ts#L13-L13

Reply:
changed that method to `parseFloat` - https://github.com/lightdash/lightdash/pull/10300/commits/75be47b7a8e74716dc57a4892e257ff49c7d5cd7 would you prefer a separate method like `getFloatFromEnvironmentVariable`?

Reply:
A separate method would be better otherwise we'll lose the current validation. i.e. I can have PGMAXCONNECTIONS=2.5

Reply:
💯